packages/xstate-wallet/src/messaging.ts

```diff
@@ -143,8 +143,15 @@ export class MessagingService implements MessagingServiceInterface {
     const {id: requestId} = request;
 
     switch (request.method) {
-      case 'WalletVersion':
-        window.parent.postMessage(jrs.success(requestId, 'xstate-wallet@VersionTBD'), '*'); // TODO: inject git or build info
+      case 'GetWalletInformation':
+        window.parent.postMessage(
```

Comment:
This should use `this.sendResponse` instead of calling `window.parent.postMessage` directly.